src/components/SEO.js

```diff
@@ -10,17 +10,31 @@ const METADATA = [
   },
 ];
 
-const DocsSiteSeo = ({ location, title }) => (
-  <SEO location={location} title={title}>
-    {METADATA.map((data) => (
-      <meta key={data.name} {...data} />
-    ))}
-  </SEO>
-);
+const DocsSiteSeo = ({ location, title, description }) => {
+  const metadata =
+    description || title
+      ? [
+          ...METADATA,
```

Comment:
Rather than sticking this one the array, I think the dynamic `meta` tags might be a little cleaner if you put it in React itself:

```js
<SEO>
  {metadata.map(...)}
  {(description || title) && <meta name='description' content={description || title} />}
</SEO>
```